iam.tf

```diff
@@ -139,7 +139,19 @@ resource "aws_security_group_rule" "minions-allow-https" {
     cidr_blocks = ["0.0.0.0/0"]
 }
 
+resource "aws_security_group_rule" "minions-allow-extra" {
+    count = "${var.enable_extra_minion_security_group}"
+    security_group_id = "${aws_security_group.minions.id}"
+
+    type = "ingress"
+    source_security_group_id = "${var.extra_minion_security_group}"
+    from_port = 30081 // Defined in default/authfe-nodeport-svc.yaml
+    to_port = 30081
```

Comment:
I really dislike this cross-repo duplication. Esp. given that this repo is notionally deployment-neutral.